### PR TITLE
Tee the logs from their screen session since scrollback has limitations.

### PR DESCRIPTION
**Pull request by ed-**
This makes this a little easier.

### CODE REVIEW

**Comment by jcru** [comment]
+1

File: scripts/redstack [hunk 1]
```diff
@@ -704,8 +704,8 @@ function init_fake_mode() {
 function cmd_start() {
     init_reddwarf_screen
 
-    screen_it api "cd $PATH_REDDWARF; bin/reddwarf-api --config-file=$USERHOME/reddwarf.conf"
-    screen_it taskmanager "cd $PATH_REDDWARF; bin/reddwarf-taskmanager --config-file=$USERHOME/reddwarf-taskmanager.conf"
+    screen_it api "cd $PATH_REDDWARF; bin/reddwarf-api --config-file=$USERHOME/reddwarf.conf | $USERHOME/reddwarf-api.log"
```
Comment (cp16net) [inline scripts/redstack]: pipe to a log file wont write it to the file dont you need to `>` redirect instead.

Comment (ed-) [inline scripts/redstack]: argh, that should be | tee file

this is what I get for hand-transplanting from the bunker

**Comment by TimSimpsonR** [comment]
+1

**Comment by cp16net** [comment]
+1

**Comment by imsplitbit** [comment]
+1

**Pull request merged**